utils.go

```diff
@@ -72,3 +72,9 @@ func logErrs(err []error) {
 		}
 	}
 }
+
+func debugLog(l string) {
```

Comment:
Yeah, just seems like a lot of extra typing.  NBD.  If you wanted variable arguments of any time, you could just do a ...interface as the last type I believe.  Anyway, could be overkill - just hate to see the same thing typed over and over :)